javav2/usecases/Creating_serverless_aurora_app/Readme.md

```diff
@@ -1,4 +1,13 @@
-#  Creating the Serverless Amazon Aurora item tracker application using the Java RdsDataClient API
+#  Creating the Serverless Amazon Aurora application using the Java RdsDataClient API
```

Comment:
Creating the Amazon Aurora Serverless application using the Java RdsDataClient API operation

Reply:
fixed